site/docs/components/status-indicator/usage.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Status Indicator
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+### When to use Status Indicator
+
+- Use the info indicator when you need to highlight general information.
+
+- Use the error indicator when communicating a critical issue that prevents the user from continuing or completing it.
+
+- Use the warning indicator to inform users of an issue or potential issue related to their current task. Use this for issues that do not prevent the user from continuing or completing their task.
+
+- Use the success indicator to confirm that a user's action has been completed successfully.
+
+### When not to use Status Indicator
+
+- Use an existing Salt component which integrates Status such as [Banner](/salt/components/banner), [Toast](/salt/components/toast) or [Tooltip](/salt/components/tooltip) where appropriate
```

Comment:
Use a status indicator where you need to accomplish something in the "When to use Status Indicator", but only if one of our other components doesn't fit your use case.

E.g. we don't want consumers to build their own custom component using a StatusIndicator, if Banner/Toast will work for their use case

Reply:
Question for design. Let's either get this merged and folow up or I can delete the `when not to use StatusIndicator` section

Reply:
You can think of StatusIndicator as a lower level component, which can be used individually by consumers if they need for their own components, but is also directly built into some of our other components. Being lower level it can be used for circumstances not covered by these other components

For example similarly in Layouts we have a Flex component, which is lower level and can be used individually, but we also have Stack and Flow layouts which use Flex internally. We would probably recommend people to use Stack/Flow if they meet the required use case, but being lower level Flex layout is a bit more flexible so can also be used individually if Stack/Flow don't work for a particular use-case